src/TimeoutMigrationTool/RavenDB/RavenDBTimeoutStorage.cs

```diff
@@ -143,11 +144,11 @@ bool filter(TimeoutData td)
         public async Task Complete()
         {
             var toolState = await TryLoadOngoingMigration();
+            var ravenToolState = (RavenToolState)toolState;
```

Comment:
This is off and I think we need to revisit why the Status has been removed from the IToolState.